doc/source/user_guide/categorical.rst

```diff
@@ -591,10 +585,8 @@ the original values:
 
     base = np.array([1, 2, 3])
 
-    try:
-        cat > base
-    except TypeError as e:
-        print("TypeError:", str(e))
+    @okexcept no_traceback  # noqa: E999
```

Comment:
what are those noqa capturing?

Reply:
the commit hook runs `flake8-rst` and it complains about syntax errors in the python code inside the rst. At first I tried to add the `:flake8-add-ignore: E999` custom role but this didn't work (same as in https://github.com/flake8-docs/flake8-rst/issues/25), so I found this workaround (see also second item in commit message). 

Another solution would have been to put the argument of the pseudodecorator in parentheses but as arguments to options in sphinx directives are added after a space on the same line I thought it might be better to stick with this convention. 

`flake8-rst` appears to be more or less abandoned (last commit in Dec 2020, no reaction to issues or pull requests for about 2 years), so I'm not sure if it's a good idea to run it in the commit hook.

Reply:
I think it'd be also fine to use `:okexcept: no_traceback` at the beginning of the ipython block, like in the other cases. But totally up to you, thanks for the info!

Reply:
I like the pseudodecorator more because I think it makes it clearer where to expect the exception if the block contains a couple of instructions